client/docs/hp_admin_crypto_client.js.html

```diff
@@ -0,0 +1,247 @@
+<!DOCTYPE html>
```

Comment:
Maybe I could set up Hydra (actual Hydra, not Nginx) to build and continuously deliver docs instead?

Reply:
That would be great. How can we contain the information about CD process in this repo?